### PR TITLE
Full screen UI improvements

### PR DESCRIPTION
**Pull request by tomasz-stefaniak**
## Description

Added history sidebar on large screens.

## Checklist

- [x] The base branch of this PR is `dev`, rather than `main`
- [ ] The relevant docs, if any, have been updated or created

## Screenshots

https://github.com/user-attachments/assets/b8fadffc-a987-4063-9d93-6e2a59a38fae

## Testing

[ For new or modified features, provide step-by-step testing instructions to validate the intended behavior of the change. ]

### CODE REVIEW

File: gui/src/context/LastSessionContext.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  setLastSessionId: () => {},
+});
+
+export const LastSessionProvider: React.FC<{
+  children: React.ReactNode;
+}> = ({ children }) => {
+  const [lastSessionId, setLastSessionId] = useState<string | undefined>();
+
+  useEffect(() => {
+    const storedId = localStorage.getItem("lastSessionId");
```
Comment (Patrick-Erichsen) [inline gui/src/context/LastSessionContext.tsx]: We have a file with typings for all of our localStorage accessors: https://github.com/continuedev/continue/blob/main/gui/src/util/localStorage.ts

So I think we should add this there and update to use the `getLocalStorage` fn.

File: gui/tailwind.config.cjs [hunk 2]
```diff
@@ -10,12 +10,15 @@ module.exports = {
   theme: {
     // Note that these breakpoints are primarily optimized for the input toolbar
     screens: {
-      xxs: "170px", // Smallest width for Primary Sidebar in VS Code
+      "2xs": "170px", // Smallest width for Primary Sidebar in VS Code
```
Comment (Patrick-Erichsen) [inline gui/tailwind.config.cjs:13]: Thanks for making this update! 😁

File: gui/src/context/LastSessionContext.tsx (hunk 1 above)
Comment (tomasz-stefaniak) [reply gui/src/context/LastSessionContext.tsx:17]: It's already there 😄 I'm just reusing an already existing value 
image

Comment (tomasz-stefaniak) [reply gui/src/context/LastSessionContext.tsx]: This code actually wasn't needed. Removed it 👌

**Review by TyDunn — CHANGES_REQUESTED** [review]
@tomasz-stefaniak Looks like a good next iteration! My two thoughts:
1. I suspect we'll want to increase the breakpoint a little, so that the history section is only added if the sidebar is expanded a bit wider than the current breakpoint
2. I think we should work on making the history page look / feel much nicer after we get this merged

**Comment by tomasz-stefaniak** [comment]
> @tomasz-stefaniak Looks like a good next iteration! My two thoughts:
> 
> 1. I suspect we'll want to increase the breakpoint a little, so that the history section is only added if the sidebar is expanded a bit wider than the current breakpoint
> 2. I think we should work on making the history page look / feel much nicer after we get this merged

1. I updated it to 1180px from 1140px. I think it's the right balance - any wider and it wouldn't open at all unless the left bar is removed.
2. I agree - I wanted to get this PR out of the way quickly but there's plenty of room for improvement on the history page UI

**Review by sestinj — APPROVED** [review]

**Pull request merged**